### DISCUSSION TITLE
Superset build issue

### DISCUSSION THREAD

**Discussion by DarwinDennis003** [apache/superset#27289] [General]
### Bug description

When I zip it from my local forked repository the build works fine. When the same done using git clone the build works fine .
Any suggestions on that?

### How to reproduce the bug

docker build -t superset .

1.[+] Building 336.8s (19/25)                                                                                                                                                   docker:default
 => [internal] load build definition from Dockerfile                                                                                                                                    0.1s
 => => transferring dockerfile: 5.96kB                                                                                                                                                  0.0s
 => [internal] load .dockerignore                                                                                                                                                       0.0s
 => => transferring context: 1.31kB                                                                                                                                                     0.0s
 => [internal] load metadata for docker.io/library/node:16-bookworm-slim                                                                                                                2.5s
 => [internal] load metadata for docker.io/library/python:3.9-slim-bookworm                                                                                                             3.1s
 => [auth] library/python:pull token for registry-1.docker.io                                                                                                                           0.0s
 => [auth] library/node:pull token for registry-1.docker.io                                                                                                                             0.0s
 => [superset-node 1/7] FROM docker.io/library/node:16-bookworm-slim@sha256:89061680359f5d9103d24f0066e2c96a8018c24d5c4fe8eb8d59d16dd4ab64ba                                            0.0s
 => [internal] load build context                                                                                                                                                       1.0s
 => => transferring context: 6.11MB                                                                                                                                                     0.9s
 => [lean  1/10] FROM docker.io/library/python:3.9-slim-bookworm@sha256:bcdcaefe092335ff0a0ed421e8a8d12b86fc2c1feb1199fbdac27d67ba808a9c                                                0.0s
 => CACHED [lean  2/10] WORKDIR /app                                                                                                                                                    0.0s
 => CACHED [lean  3/10] RUN mkdir -p /app/pythonpath superset/static superset-frontend apache_superset.egg-info requirements     && useradd --user-group -d /superset -m --no-log-init  0.0s 
 => [lean  4/10] COPY --chown=superset:superset setup.py MANIFEST.in README.md ./                                                                                                       0.1s 
 => CACHED [superset-node 2/7] RUN apt-get update -qq     && apt-get install -yqq --no-install-recommends         build-essential         python3                                       0.0s 
 => CACHED [superset-node 3/7] WORKDIR /app/superset-frontend                                                                                                                           0.0s 
 => CACHED [superset-node 4/7] RUN --mount=type=bind,target=/frontend-mem-nag.sh,src=./docker/frontend-mem-nag.sh     /frontend-mem-nag.sh                                              0.0s 
 => CACHED [superset-node 5/7] RUN --mount=type=bind,target=./package.json,src=./superset-frontend/package.json     --mount=type=bind,target=./package-lock.json,src=./superset-fronte  0.0s 
 => CACHED [superset-node 6/7] COPY ./superset-frontend ./                                                                                                                              0.0s 
 => [superset-node 7/7] RUN npm run build                                                                                                                                             332.6s 
 => => # | Processing: src/visualizations/TimeTable/index.ts
 => => # / Processing: src/visualizations/TimeTable/transformProps.ts                                                                                                                        
 => => # - Processing: src/visualizations/dashboardComponents/DashboardComponentsRegistry.ts
 => => # \ Processing: src/visualizations/dashboardComponents/ExampleComponent/ExampleComponent.tsx                                                                                          
 => => # | Processing: src/visualizations/dashboardComponents/ExampleComponent/index.ts
 => => # / Processing: src/visualizations/presets/dashboardComponents.ts                                                                                                                     
 => [lean  5/10] COPY --chown=superset:superset superset-frontend/package.json superset-frontend/                                                                                       0.1s
 => [lean  6/10] RUN --mount=type=bind,target=./requirements/local.txt,src=./requirements/local.txt     --mount=type=bind,target=./requirements/development.txt,src=./requirements/de 

### Superset version

3.1.0

### Python version

3.9

### Node version

18 or greater

### Browser

Chrome

### Checklist

- [X] I have searched Superset docs and Slack and didn't find a solution to my problem.
- [X] I have searched the GitHub issue tracker and didn't find a similar bug report.
- [ ] I have checked Superset's logs for errors and if I found a relevant Python stacktrace, I included it here as text in the "additional context" section.

**Comment by rusackas** [comment]
Sorry, I'm probably just misunderstanding, but if it works on your fork, and it works when you get clone, can you tell us where it's _not_ working so we can try to replicate the issue?

**Comment by sfirke** [comment]
> Sorry, I'm probably just misunderstanding, but if it works on your fork, and it works when you get clone, can you tell us where it's _not_ working so we can try to replicate the issue?

I am wondering this too.  Also "Superset build issue" is a very broad title, if you could be more specific that would enhance the bug tracker.

**Comment by DarwinDennis003** [comment]
I have forked the superset repository version SUPERSET_VERSION=**1.5.1**  got this from Readme.md.
Machine : Windows 
I am using node version of **20.10.0**
git clone from local repository --> cd superset-frontend --> npm i 
when i run npm run build 

![image](https://github.com/apache/superset/assets/122859676/008fe4a5-ee5c-4635-8a7b-b12c587abbe4)

Somehow i thought of Zip download the repo instead of Cloning from my local repository , 
Surprisingly , the build completed with some minor code errors at the end.

![image](https://github.com/apache/superset/assets/122859676/cb1742bb-3857-4395-bf90-f7611e84704c)

Waiting for your replies , thanks !

**Comment by michael-s-molina** [comment]
Moving this to a discussion given that neither Windows or version 1.5 are supported anymore. Highly recommend upgrading to a supported version.

**Reply by duniang818** [reply]
maybe i have the same issue with the above.
when i run : npm run build  
but it stop at this file: D:\superset-fork\superset-frontend\src\visualizations\TimeTable\transformProps.ts

![image](https://github.com/apache/superset/assets/2695936/81c9bcfc-7ddb-4984-bcd5-37b799a156e0)
so ,how to solve this issue, it is very diffculty for me, can someone help me?
it is a long time i have hung at the frontend code, i want to add some charts to it, but now i can't.
Although the backend can run, that just use the normal function, but i deeply want to change some charts. thanks very much who help me.

**Reply by lokeshMrudhul** [reply]
I'm facing the same issue

![image](https://github.com/user-attachments/assets/429ff929-3383-48a8-9c74-0b5077d0cd76)

**Reply by emosonic** [reply]
How to fix that? My loading stops at this file:
# / Processing: src/visualizations/presets/dashboardComponents.ts

**Reply by DanielViniAssis** [reply]
I facing the same issue, can someone help?
